src/billing/components/Free/OrgLimitStat.tsx

```diff
@@ -31,26 +35,24 @@ const getStat = (name: string, value: any) => {
       const mbPerSecond = kbToMb(value)
       const mb5Min = floor(mbPerSecond * sIn5Min)
       return `${mb5Min} MB / 5 min`
-    case 'bucket':
-      return `${nsToDays(value)} days`
+    case 'maxRetentionSeconds':
+      return `${secondsToDays(value)} days`
```

Comment:
this an intentional change from nanoseconds to seconds?

Reply:
Yup, the data is coming in from IDPE and not Quartz. IDPE returns seconds from the limits API, Quartz returns nanoseconds